hammr/utils/hammr_utils.py

```diff
@@ -240,3 +240,31 @@ def create_user_ssh_key(api, login, sshKey):
         printer.out("Impossible to create sshKey ["+mySshKey.name+"]", printer.ERROR)
         return 2
     return key
+
+def is_appliance_uri(image_uri):
```

Comment:
As the method aims to be generic, there is a problem with "users/guest/appliances/205" for example because it's an appliance URI that will not be considered as one.
And after reflection, maybe the method should be named something like "contains_appliance_uri" or "is_uri_based_on_appliance" because an image uri based on an appliance will return true.
And the parameter "image_uri" may be renamed too.